Mathlib/CategoryTheory/ConnectedComponents.lean

```diff
@@ -40,6 +40,18 @@ def ConnectedComponents (J : Type u₁) [Category.{v₁} J] : Type u₁ :=
 instance [Inhabited J] : Inhabited (ConnectedComponents J) :=
   ⟨Quotient.mk'' default⟩
 
+/-- Every function from connected components of a category gives a functor to discrete category -/
+def ConnectedComponents.connectedToDiscrete.{u_1, u_2} {X C : Type u_1} [Category.{u_2, u_1} C]
```

Comment:
- I don't think `C` and `X` need to be in the same universe. You can use `Type*` and drop the universe annotation on the `Category` instance.
- The name doesn't really make sense. Maybe `ConnectedComponents.functorToDiscrete`?